src/main/resources/assets/jei/lang/de_de.lang

```diff
@@ -49,60 +49,60 @@ config.jei.title=%MODNAME-Konfiguration
-config.jei.advanced.colorSearchEnabled=Aktiviere Farbsuche
-config.jei.advanced.colorSearchEnabled.comment=Suche nach Gegenständen mittels ihrer Farbe und zeige die suchbaren Farben in Tooltips in der Item-Liste.
-config.jei.advanced.debugModeEnabled=Aktiviere Debug-Modus
-config.jei.advanced.debugModeEnabled.comment=Fügt tausende Test-Items und einige Debug-Rezepte hinzu. Nur für Entwickler bestimmt.
+config.jei.advanced.itemBlacklist.comment=Items, die nicht angezeigt werden sollen. Format: modId[:name[:meta]]. Der Listeneditiermodus entfernt und fügt automatisch neue Items zu dieser Liste hinzu.
+config.jei.advanced.colorSearchEnabled=Aktiviere die Farbsuche
+config.jei.advanced.colorSearchEnabled.comment=Suche nach Items mittels ihrer Farbe und zeige die suchbaren Farben in der Schnellinfo(Tooltip) in der Items-Liste.
+config.jei.advanced.debugModeEnabled=Aktiviere den Fehlersuche-Modus
+config.jei.advanced.debugModeEnabled.comment=Fügt tausende Test-Items und einige Fehlersuche-Rezepte hinzu. Nur für Entwickler bestimmt.
 config.jei.advanced.centerSearchBarEnabled=Zentriere Suchleiste
 config.jei.advanced.centerSearchBarEnabled.comment=Die JEI-Suchleiste wird unten in der Mitte des Bildschirms angezeigt.
-config.jei.advanced.modNameFormat=Mod-Namensformat
-config.jei.advanced.modNameFormat.comment=Wie der Mod-Name in Tooltips für JEI-Interfaces formatiert werden soll. Zum Deaktivieren leer lassen.
-config.jei.advanced.maxColumns=Max Width
-config.jei.advanced.maxColumns.comment=The maximum width of the ingredient list.
-config.jei.advanced.giveMode=Give Mode
-config.jei.advanced.giveMode.comment=Choose if JEI should give items direct to the inventory (inventory) or pick them up with the mouse (mouse_pickup).
+config.jei.advanced.modNameFormat=Modifikations-Namensformat
+config.jei.advanced.modNameFormat.comment=Wie der Modifikations-Name in der Tooltip für die JEI-Oberfläche formatiert werden soll. Zum Deaktivieren einfach leer lassen.
+config.jei.advanced.maxColumns=Maximale Breite
+config.jei.advanced.maxColumns.comment=Die maximale Breite der Zutaten-Liste.
+config.jei.advanced.giveMode=Geben-Modus
+config.jei.advanced.giveMode.comment=Wähle ob JEI die Items direkt in den Inventar geben soll oder diese aufnimmt mit der Maus.
 
 # Hide Ingredients Mode
 gui.jei.editMode.description=JEI-Listeneditiermodus:
 gui.jei.editMode.description.hide=Ausblenden mit %CTRL-Klick.
-gui.jei.editMode.description.hide.wild=Ausblenden per Wildcard mit %CTRL-Rechtsklick.
+gui.jei.editMode.description.hide.wild=Ausblenden per einem Platzhalter mit %CTRL-Rechtsklick.
 
 # PC/Linux Control Key
 key.jei.ctrl=STRG
 # Mac Command Key
 key.jei.ctrl.mac=CMD
 
 # Recipe Categories
-gui.jei.category.craftingTable=Crafting-Rezepte
+gui.jei.category.craftingTable=Herstellungs-Rezepte
```

Comment:
As this is crafting specifically, I don't quite like this change. However, I also don't like the original. How about this?
```suggestion
gui.jei.category.craftingTable=Fertigungsrezepte
```